MoreLinq/MoreLinq.csproj

```diff
@@ -115,7 +115,8 @@
     <VersionPrefix>3.2.0</VersionPrefix>
     <Authors>MoreLINQ Developers.</Authors>
     <TargetFrameworks>net451;netstandard1.0;netstandard2.0</TargetFrameworks>
-    <LangVersion>7.3</LangVersion>
+    <LangVersion>8</LangVersion>
+    <NullableContextOptions>enable</NullableContextOptions>
```

Comment:
This should be `<Nullable>enabled</Nullable>` now

Reply:
Fixed by @Orace in 9723435a0f0318d90ccec7085a15c24babcf0f4f.